README.md

```diff
@@ -26,6 +26,23 @@ On Windows, there are binary wheel you can install without MySQLConnector/C or M
 `sudo yum install python3-devel `  # Red Hat / CentOS
 
 `brew install mysql-connector-c`  # macOS (Homebrew)
+#### Note on macOS
+homebrew's `mysql-connector-c` may have incorrect configuration options that cause compilation errors.  Under the `#create options` comment on or about line 112 of `/usr/local/bin/mysql_config`: 
```

Comment:
The issue (MySQL's bug) is not homebrew specific.

Reply:
I'll update to be more general.